app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -1,4 +1,5 @@
 class GetUspsProofingResultsJob < ApplicationJob
+  MILLISECONDS_PER_SECOND = 1000.0 # Specify float value to use floating point math
```

Comment:
Do you think it'd be any clearer (i.e. no need for the comment) if we were explicit in converting this to a float?

```suggestion
  MILLISECONDS_PER_SECOND = 1000.to_f
```

Or doing this at the point where we're doing the precision math?

```rb
    request_delay_in_seconds = IdentityConfig.store.
      get_usps_proofing_results_job_request_delay_milliseconds / MILLISECONDS_PER_SECOND.to_f
```

Reply:
Personally I think the comment is preferable because it explains the intent of something that at first seems unnecessary. If someone later removes it thinking it's unnecessary it'll revert to integer math and do weird stuff eg `750/1000 = 0`

Reply:
Fair enough (and I agree with that concern!) 👍